src/platform/plugins/private/presentation_panel/public/panel_component/panel_header/presentation_panel_title.tsx

```diff
@@ -94,78 +32,85 @@ export const PresentationPanelTitle = ({
   panelDescription?: string;
   viewMode?: ViewMode;
 }) => {
-  const [panelTitleElmRef, setPanelTitleElmRef] = useState<HTMLElement | null>(null);
+  const { euiTheme } = useEuiTheme();
+
+  const onClick = useCallback(() => {
+    openCustomizePanelFlyout({
+      api: api as CustomizePanelActionApi,
+      focusOnTitle: true,
+    });
+  }, [api]);
+
   const panelTitleElement = useMemo(() => {
     if (hideTitle) return null;
-    const titleClassNames = classNames('embPanel__titleText', {
-      // eslint-disable-next-line @typescript-eslint/naming-convention
-      embPanel__placeholderTitleText: !panelTitle,
-    });
+
+    const titleStyles = css`
+      ${euiTextTruncate()};
+      font-weight: ${euiTheme.font.weight.bold};
+
+      .kbnGridPanel--active & {
+        pointer-events: none; // prevent drag event from triggering onClick
+      }
```

Comment:
This CSS gives the same behaviour as `usePresentationPanelTitleClickHandle` but without the over complicated logic